src/components/layout/WithVehicleReference.tsx

```diff
@@ -1,60 +1,39 @@
-import React, { FC, PropsWithChildren } from 'react';
+import React, { FC, ReactNode } from 'react';
 
-import { GridItem, Heading } from '@chakra-ui/react';
-
-import BaseGridLayout, { repeatArea } from './BaseGrid';
+import TwoColumsLayout from './TwoColumnsLayout';
 import VehicleReference, { VehicleReferenceProps } from '../vehicleReference';
 import Stack from '../stack';
-import Link from '../link';
-import { ArrowLeftIcon } from '../icons';
 
 interface Props {
-  title: string;
+  title?: string;
   backLink?: {
     text: string;
     url: string;
   };
   vehicle: VehicleReferenceProps;
+  header: ReactNode;
+  leftContent: ReactNode;
```

Comment:
Can we keep `children` here? It makes sense to use `children` if we're only interested in one type of content